packages/wp-now/src/get-wp-cli-path.ts

```diff
@@ -0,0 +1,12 @@
+import path from 'path';
+import getWpNowPath from './get-wp-now-path';
+
+/**
+ * The path for wp-cli phar file within the WP Now folder.
+ */
+export default function getWpCliPath() {
+	if (process.env.NODE_ENV !== 'test') {
+		return path.join(getWpNowPath(), 'wp-cli.phar');
+	}
+	return path.resolve('./wp-cli.phar');
```

Comment:
Should this be a temp directory?

Reply:
I found that using a temporal directory on my laptop, the `wp-cli.phar` cannot be executed.
I tried setting the right permissions `fs.chmod()` to the file and folder and it didn't work.

Reply:
@sejas Ok. I think we should use a tmp directory in any case. I added with 5e8e97ce

@adamziel Any idea why the test might fail?

Reply:
@danielbachhuber , with the temp folder, it seems to pass ok in CI. But it is still failing on my local computer.
Maybe it's something just in my environment.

Reply:
@sejas It failed locally for me too.

Reply:
> Is it getting mounted correctly @sejas? What do you see when you list files?

Yes, the file is accessible in VFS and the file size is correct. It must be something related to MacOs and the tmp path.

Reply:
@sejas would you start a new issue for this? I'd like to make sure this works on MacOs